interface.go

```diff
@@ -54,9 +67,14 @@ type Config struct {
 	// Warning: This API should not be considered stable and will change soon.
 	Versions []protocol.VersionNumber
 	// Ask the server to truncate the connection ID sent in the Public Header.
+	// If not set, the default checks if
 	// This saves 8 bytes in the Public Header in every packet. However, if the IP address of the server changes, the connection cannot be migrated.
 	// Currently only valid for the client.
 	RequestConnectionIDTruncation bool
+	// If true is returned, a source address validation is performed
```

Comment:
It's uncommon to have a verify function return true if stuff is wrong (verbs are usually imperative in function names). Maybe rename to AcceptSTK, IsValidSTK, CheckSTK and invert the return value?